File: tests/fixtures/source-windows/scala/package-class.scala
package windows

final case class ReportEntry(title: String, owner: Option[String])

/** Keep report entries in insertion order. */
final class ReportBook:
  private var entries = Vector.empty[ReportEntry]

  def add(title: String, owner: Option[String]): Unit =
    entries = entries :+ ReportEntry(title, owner)

  def render: Vector[String] =
    entries.map { entry =>
      val owner = entry.owner.getOrElse("unassigned")
      s"${entry.title} — $owner"
    }

  def size: Int = entries.size

File: tests/fixtures/source-windows/kotlin/package-class.kt
package windows

data class ReportEntry(val title: String, val owner: String?)

/** Keep report entries in insertion order. */
class ReportBook {
    private val entries = mutableListOf<ReportEntry>()

    fun add(title: String, owner: String?) {
        entries += ReportEntry(title, owner)
    }

    fun render(): List<String> = entries.map { entry ->
        val owner = entry.owner ?: "unassigned"
        "${entry.title} — $owner"
    }

    val size: Int
        get() = entries.size
}
